test/unit/specs/components/staking/TabValidators.spec.js

```diff
@@ -1,49 +1,95 @@
-import setup from "../../../helpers/vuex-setup"
+import { shallowMount } from "@vue/test-utils"
 import TabValidators from "renderer/components/staking/TabValidators"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
 
-const { stakingParameters } = lcdClientMock.state
+// we don't show data of a validator in this component so we just provide stubs here
+const delegates = new Array(3).fill({})
 
 describe(`TabValidators`, () => {
-  let wrapper
-  let store
-  const { mount } = setup()
+  let wrapper, $store
+
+  const getters = {
+    delegates: {
+      delegates,
+      loading: false,
+      loaded: true
+    },
+    connected: true
+  }
 
   beforeEach(async () => {
-    const instance = mount(TabValidators, {
-      doBefore: ({ store }) => {
-        store.commit(`setConnected`, true)
-      },
-      stubs: {
-        "tm-data-connecting": true,
-        "tm-data-loading": true,
-        "tm-data-empty": true,
-        "short-bech32": true
+    $store = {
+      getters
+    }
+
+    wrapper = shallowMount(TabValidators, {
+      mocks: {
+        $store
       }
     })
-    wrapper = instance.wrapper
-    store = instance.store
-    store.state.stakingParameters = stakingParameters
-    await store.dispatch(`getDelegates`)
   })
 
   it(`has the expected html structure`, async () => {
```

Comment:
maybe we should stop using this as a description because it is really unhelpful. 

"should display the validator table" would be much better in this case.

Reply:
i opened a ticket to discuss this further. not required for this PR.